src/js/select2/dropdown/search.js

```diff
@@ -9,9 +9,9 @@ define([
 
     var $search = $(
       '<span class="select2-search select2-search--dropdown">' +
-        '<input class="select2-search__field" type="search" tabindex="-1"' +
+        '<input class="select2-search__field" type="text" tabindex="-1"' +
```

Comment:
The rest of the changes make sense, but I can't figure out why this change was needed. Can a search field not function as a combobox?